src/genesis/helpers/field_enums.py

```diff
@@ -32,10 +33,15 @@ def select_query(cls, tables: List[str] = None, prefix=False) -> str:
 
     @classmethod
     def select_where(
-        cls, where_clause: str, tables: List[str] = None, prefix=False
+        cls, where_clause: str, tables: Optional[List[str]] = None, prefix=False
     ) -> str:
         return f"{cls.select_query(tables=tables, prefix=True)} WHERE {where_clause}"
 
+    @classmethod
+    @abstractmethod
+    def get_table(cls):
+        pass
```

Comment:
Combination of `@abstractmethod`, `@classmethod` and `@property` is problematic
I haven't found other solution than just transform it into abstract class method.